python/tests/test_session.py

```diff
@@ -139,10 +144,10 @@ def test_failing_case(self):
 
         session.stop(False)
 
-    def test_update_method(self):
+    def test_raises_exception_if_result_value_not_used(self):
         session = SauceSession()
 
         session.start()
 
-        session.update_test_result(True)
-        session.stop()
+        with pytest.raises(TypeError):
```

Comment:
@titusfortner this test is now failing, but the logic looks ok.

Reply:
Heh, I accidentally changed the call to `session.stop(True)` which of course won't raise the error.